src/DynamicImage/widget/DynamicImage.js

```diff
@@ -96,8 +96,7 @@ define([
                 this._setToDefaultImage();
             }
 
-            if (callback)
-                callback();
+            callback && callback();

```

Comment:
This is dirty code. We use in most of our widgets 
```javascript 
mendix.lang.nullExec(callback);
``` 
to execute this callback.